code/modules/ruins/spaceruin_code/oldstation.dm

```diff
@@ -29,12 +29,15 @@
 	will form the backbone of our Energy weapon catalogue.<br><br>The K14 is expected to undergo revision to fix the ammo issues, the K15 is expected to replace the 'stun' setting with a \
 	'disable' setting in an attempt to bypass the ammo issues."
 
+/**
+ * Supermatter crystal fluff paper used in Charlie station ruin
+ */
 /obj/item/paper/fluff/ruins/oldstation/protosing
```

Comment:
could you update the path used here as well?

Reply:
done